### ISSUE TITLE
Bug when trying to add postal address and PIN (based on "imamopravoznati-theme-0.39.1.6" branch)

### ISSUE THREAD

**Issue by schlos** [codeforcroatia/imamopravoznati-theme#129]
**Describe the bug**

When testing "imamopravoznati-theme-0.39.1.6" branch: 
https://github.com/codeforcroatia/imamopravoznati-theme/tree/imamopravoznati-0.39.1.6

**1) When trying to add Postal Address:**

```
An ActionView::Template::Error occurred in user#signchangeaddress:

  undefined method `address' for #<User:0x0000561d727c1ad8>
  lib/themes/imamopravoznati-theme/lib/views/user/signchangeaddress.html.erb:14:in `block in _lib_themes_imamopravoznati_theme_lib_views_user_signchangeaddress_html_erb__1875958028824275381_47342237364060'


-------------------------------
Request:
-------------------------------

  * URL        : https://imamopravoznati.org/profile/change_address
  * HTTP Method: GET
  * IP address : xxxxxxxxxxxx
  * Parameters : {"controller"=>"user", "action"=>"signchangeaddress"}
  * Timestamp  : 2022-01-20 20:07:32 +0100
  * Server : imamopravoznati.org
    * Rails root : /data/www/imamopravoznati.org/alaveteli
  * Process: 43174
```

**2) When trying to add PIN - Personal Identification Number:**

```
An ActionView::Template::Error occurred in user#signchangepin:

  undefined method `national_id_number' for #<User:0x0000561d718bda50>
  lib/themes/imamopravoznati-theme/lib/views/user/signchangepin.html.erb:14:in `block in _lib_themes_imamopravoznati_theme_lib_views_user_signchangepin_html_erb___397204452511509440_47342229499580'


-------------------------------
Request:
-------------------------------

  * URL        : https://imamopravoznati.org/profile/change_pin
  * HTTP Method: GET
  * IP address : xxxxxxxxxxxx
  * Parameters : {"controller"=>"user", "action"=>"signchangepin"}
  * Timestamp  : 2022-01-20 19:53:22 +0100
  * Server : imamopravoznati.org
    * Rails root : /data/www/imamopravoznati.org/alaveteli
  * Process: 43174
```

**To Reproduce**

**Expected behavior**

**Screenshots**

**Desktop (please complete the following information):**
 - OS: [e.g. Windows]

**Smartphone (please complete the following information):**

**Additional context**

The "imamopravoznati-theme-0.39.1.6" branch is to add support for:
- Customize request email - add postal address and PIN (OIB) #12

**Comment by schlos** [comment]
May be related to:
* Post install script was not run - Directly inheriting from ActiveRecord::Migration is not supported. ("imamopravoznati-theme-0.39.1.6" branch) #130

**Comment by schlos** [comment]
Fixed via #130
